aiida/storage/psql_dos/migrator.py

```diff
@@ -312,6 +306,27 @@ def initialise_database(self) -> None:
             context.stamp(context.script, 'main@head')
             self.connection.commit()
 
+    def delete_tables(self, exclude_tables: list[str] | None = None) -> None:
+        """Delete the tables of the database.
```

Comment:
Maybe mention that these are only the tables from the AiiDA database schema? [1]

[1] I guess that is still the case? What happens, if there is an additional "foreign-created" table in the DB?

Reply:
That should be untouched as we deduce the tables by introspecting our own schema. If there are other tables, they should not be affected in principle. Will clarify this in the docstring